src/System.Runtime.Caching/src/System.Runtime.Caching.csproj

```diff
@@ -0,0 +1,105 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.Runtime.Caching</AssemblyName>
+    <ProjectGuid>{A7B6FB6E-F484-42D7-8A5E-F7D0DCC03829}</ProjectGuid>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Although we have a netstandard configuration, we know we are not currently UAP compatible-->
+    <UWPCompatible>false</UWPCompatible>
+    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsWindows)' != 'true'">true</GeneratePlatformNotSupportedAssembly>
```

Comment:
Could you please instead use `GeneratePlatformNotSupportedAssemblyMessage` and pass in a custom SR. message that is the pattern we've been using in the other packages:

https://github.com/safern/corefx/blob/0c1541a5a1d8fe75a80ba9484f5bebfbf9b06583/src/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj#L11

Reply:
@KKhurin you can add this since you're wiring up the resources atm.

Reply:
Done!